app/services/idv/steps/ssn_step.rb

```diff
@@ -23,11 +23,8 @@ def call
         )
 
         idv_session.delete('applicant')
-        # rubocop:disable Style/IfUnlessModifier
-        if IdentityConfig.store.doc_auth_verify_info_controller_enabled
-          exit_flow_state_machine
-        end
-        # rubocop:enable Style/IfUnlessModifier
+
+        flow_session[:flow_path] = @flow.flow_path
```

Comment:
When the SsnController is extracted, we'll need a more general way to put the flow_path in the session or otherwise get access to it. Maybe it needs to be added to the session from the beginning rather than being set on the flow.